src/components/slider/slider.spec.ts

```diff
@@ -0,0 +1,440 @@
+import {
+    it,
+    describe,
+    beforeEach,
+    beforeEachProviders,
+    inject,
+    async,
+} from '@angular/core/testing';
+import {TestComponentBuilder, ComponentFixture} from '@angular/compiler/testing';
+import {Component, DebugElement, provide, ViewEncapsulation} from '@angular/core';
+import {By} from '@angular/platform-browser';
+import {MdSlider} from './slider';
+import {HAMMER_GESTURE_CONFIG} from '@angular/platform-browser';
+import {TestGestureConfig} from './test-gesture-config';
+
+describe('MdSlider', () => {
+  let builder: TestComponentBuilder;
+  let gestureConfig: TestGestureConfig;
+
+  beforeEachProviders(() => [
+    provide(HAMMER_GESTURE_CONFIG, {useFactory: () => {
+      gestureConfig = new TestGestureConfig();
+      return gestureConfig;
+    }})
+  ]);
+
+  beforeEach(inject([TestComponentBuilder], (tcb: TestComponentBuilder) => {
+    builder = tcb;
+  }));
+
+  describe('standard slider', () => {
+    let fixture: ComponentFixture<StandardSlider>;
+    let sliderDebugElement: DebugElement;
+    let sliderNativeElement: HTMLElement;
+    let sliderInstance: MdSlider;
+    let testComponent: StandardSlider;
+
+    beforeEach(async(() => {
+      builder.createAsync(StandardSlider).then(f => {
+        fixture = f;
+        fixture.detectChanges();
+
+        testComponent = fixture.debugElement.componentInstance;
+
+        sliderDebugElement = fixture.debugElement.query(By.directive(MdSlider));
+        sliderNativeElement = sliderDebugElement.nativeElement;
+        sliderInstance = sliderDebugElement.injector.get(MdSlider);
```

Comment:
You should be able to just say `sliderDebugElement.componentInstance` here. You only need to use `injector.get` when you're getting a `@Directive` instead of a `@Component`